### DISCUSSION TITLE
面板授权访问似乎不支持IPv6

### DISCUSSION THREAD

**Discussion by ravizhan** [natfrp/wiki#147] [使用问题]
在地址栏选择`使用本机 IP`时，会出现如下错误
![image](https://user-images.githubusercontent.com/37010563/185837062-d12a29e7-194b-454b-963c-e35cf0bd16dc.png)
![image](https://user-images.githubusercontent.com/37010563/185836926-831d93c8-80ce-47c6-acaf-e9da1b6136d8.png)
然后发现与面板连接时使用的是IPv6，禁用IPv6之后就没事了

**Comment by tidymoon** [comment]
按照之前管理员的说法 有ipv6了还要内网穿透干嘛 所以支持ipv6没意义
